docs/reference/index-modules/indexing-pressure.asciidoc

```diff
@@ -54,6 +54,20 @@ overhead (buffers, listeners, etc) which also require heap space. Other
 components of {es} also require memory. Setting this limit too high can deny
 operating memory to other operations and components.
 
+[discrete]
+[[replica-resilience]]
+=== Replica resilience
+
+When a replica is unable to process a new indexing request, the replica is marked
```

Comment:
This suggest that the replica has received the indexing request. I wonder if this should be restated as "Replication resilience", i.e. the process of replicating the data from the primary to the replica.